_config.yml

```diff
@@ -1,6 +1,6 @@
 name: Ruby Conferences
 url: https://rubyconferences.org
-safe: true
+safe: false
```

Comment:
> Plugins in the _plugins directory (or its equivalent(s)) will not be loaded when Jekyll is running in safe mode. 
https://jekyllrb.com/docs/plugins/installation/

Since we have our own `_plugins/continent_tag.rb` we need to set this to `false`.